src/test/java/teammates/test/cases/action/AdminActivityLogPageActionTest.java

```diff
@@ -1,27 +1,566 @@
 package teammates.test.cases.action;
 
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
 import org.testng.annotations.Test;
 
+import teammates.common.util.ActivityLogEntry;
 import teammates.common.util.Const;
+import teammates.common.util.JsonUtils;
+import teammates.common.util.TimeHelper;
+import teammates.test.driver.FileHelper;
+import teammates.test.driver.TestProperties;
 import teammates.ui.controller.AdminActivityLogPageAction;
+import teammates.ui.controller.AjaxResult;
+import teammates.ui.controller.ShowPageResult;
+import teammates.ui.pagedata.AdminActivityLogPageData;
+import teammates.ui.pagedata.PageData;
+
+import com.google.gson.reflect.TypeToken;
 
 public class AdminActivityLogPageActionTest extends BaseActionTest {
 
+    // typicalLogMessage.json will give a List<List<String>>. These constant are indexes
```

Comment:
No need to explicitly mention "typicalLogMessage.json". "The test data" will do just fine and more future-proof in case you rename the JSON file.